### ISSUE TITLE
[Dropdown Menu] Flicker before dropdown JS loads - fixed

### ISSUE THREAD

**Issue by shaunmorrison** [zurb/foundation-sites#7970]
The submenu classes `.dropdown.menu .submenu` hide the submenus – unfortunately all the submenus show for a brief sub-second moment on page load before the JS classes are written (creating an ugly re-jiggle of page elements). You can fix this with:
```
.nav-root{
    >li{
         >ul{
             display:none;
        }
    }
}
```
If you have Responsive Navigation you can specify `@include breakpoint(large) { .nav-root {etc} }` to ensure it doesn't mess with the Drilldown Navigation. If you are concerned about accessibility you could have a `<noscript>file.css</noscript>` overriding the `display:none` here.

**Comment by zurbchris** [comment]
This issue was addressed with 63f0242 , and you shouldn't be seeing a flicker if you're loading in the style sheets before the script. The script removes the `.no-js` class after loading. 

```css
  .no-js & ul {
      display: none;
    }
```

**Comment by gakimball** [comment]
Yep, and our [starter templates](http://foundation.zurb.com/sites/docs/starter-projects.html) include the `.no-js` class in the `<html>` tag, so this kicks in before the JavaScript is done loading.

If you're integrating Foundation into your own project, add the `.no-js` class to the `<html>` tag, so that components with no-JS fallbacks can function properly.
